notebooks/06_Serving/seldondeployment.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: machinelearning.seldon.io/v1
```

Comment:
Can you clean up the file names? 
You have two two files. 

- seldondeployment
- canary

Reply:
Addressed.